src/vstart.sh

```diff
@@ -443,6 +443,16 @@ EOF
         log file = $CEPH_OUT_DIR/\$name.\$pid.log
         admin socket = $CEPH_OUT_DIR/\$name.\$pid.asok
 
+EOF
+	for rgw in `seq 0 $((CEPH_NUM_RGW-1))`
+	do
+	    wconf <<EOF
+[client.rgw.$rgw]
+        rgw frontends = $rgw_frontend port=$((CEPH_RGW_PORT + rgw * 10))
+
+EOF
```

Comment:
a similar approach came up last year, and i suggested passing `rgw_frontends` on the command line in `do_rgw()`, instead of setting it in the config. that would allow you to create a vstart cluster with -n, and then later increase `CEPH_NUM_RGW` without having to rewrite ceph.conf. see https://github.com/ceph/ceph/pull/8241#issuecomment-220417387. do you think that would work here?